src/features/singleRight/delegate/ChooseServicePage/ChooseServicePage.tsx

```diff
@@ -173,16 +174,30 @@ export const ChooseServicePage = () => {
     void dispatch(removeServiceResource(identifier));
   };
 
+  const localizeNotDelegableCode = (notDelegableCode: string | undefined) => {
```

Comment:
The name suggests this function will localize the error code into local language versions, but it doesn't actually do that.
Perhaps it could do the fetching from the language files as well and return that as a string?
Something like this:

let errorCode : string;
switch-case block to set errorCode;
return t(`single_rights.${errorCode}_title`;

Reply:
It returns the key for the different localizations so your suggestion will break when we do t(single_rights.${errorCode}

Reply:
Yeah, but then you can just sent the result from the method directly into the prop, since it's already localized (as the name suggests)

Right now, it looks like you have to localize it twice (since t('......') is where the localizaion really happens,) because the method doesn't actually do what it says it does.

Alternatively you can rename the method into something like, getErrorReasonTextKey

Reply:
I think you should study the new code in vs code, it's a new solution I do now than before. Sending it directly into the prop is exactly what I do.

I'll change the method name though

Reply:
Ahh now I understand what you mean, can change it a bit then. Btw it's probably to get points through by suggesting changes in github then people can see what you mean in code instead of having to imagine the code on what you write.